packages/realm/src/Realm.ts

```diff
@@ -1601,149 +1479,85 @@ export declare namespace Realm {
-      /** @deprecated Please use named imports */
       export type MongoDBCollection<T extends Document> = MongoDBCollectionType<T>;
-      /** @deprecated Please use named imports */
       export type NewDocument<T extends Document> = NewDocumentType<T>;
-      /** @deprecated Please use named imports */
       export type OperationType = OperationTypeType;
-      /** @deprecated Please use named imports */
       export type RenameEvent = RenameEventType;
-      /** @deprecated Please use named imports */
       export type ReplaceEvent<T extends Document> = ReplaceEventType<T>;
-      /** @deprecated Please use named imports */
       export type Update = UpdateType;
-      /** @deprecated Please use named imports */
       export type UpdateDescription = UpdateDescriptionType;
-      /** @deprecated Please use named imports */
       export type UpdateEvent<T extends Document> = UpdateEventType<T>;
-      /** @deprecated Please use named imports */
       export type UpdateOptions = UpdateOptionsType;
-      /** @deprecated Please use named imports */
       export type UpdateResult<IdType> = UpdateResultType<IdType>;
     }
   }
 
-  /** @deprecated Please use named imports */
   // eslint-disable-next-line @typescript-eslint/no-namespace
   export namespace Types {
-    /** @deprecated Please use named imports */
     export type Bool = boolean;
-    /** @deprecated Please use named imports */
     export type String = string;
-    /** @deprecated Please use named imports */
     export type Int = number;
-    /** @deprecated Please use named imports */
     export type Float = number;
-    /** @deprecated Please use named imports */
     export type Double = number;
-    /** @deprecated Please use named imports */
     export type Decimal128 = Realm.BSON.Decimal128;
-    /** @deprecated Please use named imports */
     export type ObjectId = Realm.BSON.ObjectId;
-    /** @deprecated Please use named imports */
     export type UUID = Realm.BSON.UUID;
-    /** @deprecated Please use named imports */
     export type Date = GlobalDate;
-    /** @deprecated Please use named imports */
     export type Data = ArrayBuffer;
-    /** @deprecated Please use named imports */
     export type List<T> = Realm.List<T>;
-    /** @deprecated Please use named imports */
     export type Set<T> = Realm.Set<T>;
-    /** @deprecated Please use named imports */
     export type Dictionary<T> = Realm.Dictionary<T>;
-    /** @deprecated Please use named imports */
     export type Mixed = unknown;
-    /** @deprecated Please use named imports */
     export type LinkingObjects<ObjectTypeT, LinkingPropertyName> = Realm.Results<ObjectTypeT>;
   }
 }
 
 // Exporting a deprecated global for backwards compatibility
 const RealmConstructor = Realm;
 declare global {
-  /** @deprecated Will be removed in v13.0.0. Please use named imports */
+  /** @deprecated Will be removed in v13.0.0. Please use a regular import. */
```

Comment:
Should we say "default import" here instead?

Reply:
I reformulated this as "Please use an import statement."